azurelinuxagent/distro/default/osutil.py

```diff
@@ -261,6 +261,9 @@ def set_selinux_context(self, path, con):
         Returns exit result.
         """
         if self.is_selinux_system():
+            if not os.path.exists(path):
```

Comment:
Fair, we discussed this with Brendan a bit as well. There are arguments for both, but we can live with these changes. So LGTM.